TagsCheck.py

```diff
@@ -1117,7 +1136,12 @@ def check_summary(self, pkg, lang, ignored_words):
 '''This rpm requires a specific release of another package.''',
 
 'no-url-tag',
-'''The URL tag is missing.''',
+'''The URL tag is missing. Please add a http or ftp link to the project location.''',
+
+'no-pkg-config-provides',
+'''The package installs a .pc file but does not provide pkgconfig(..) provides.
+The most likely reason for that is that it was built without BuildRequires: pkg-config.
```

Comment:
`BuildRequires: /usr/bin/pkg-config`

There are multiple compatible implementations of `pkg-config` binary.

Reply:
At least in openSUSE we have provides, path requires should be last resort no?

Reply:
It depends. At least in this case, RPM itself generates `Requires: /usr/bin/pkg-config` with the pkgconfig dep generator for this reason.

It's generally consistent to just request the binary, since the package name does in fact differ across distros.

Reply:
Yeah but both fedora and openSUSE state that it should be either BR: pkgconfig or BR: pkg-config. So we should use what's in those wikies

Reply:
We don't actually have anything that mandates `BR: pkgconfig` in Fedora, because we include it in our buildroot.